src/app/Dashboard/RecentResourceItem.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+// import { Link } from 'react-router-dom';
+
+
+export default function RecentResourceItem(props) {
+	let TopicItem;
+
+	/**
+	 * This function passes in a topics array and item object from a higher level component
+	 * a forEach loop is called on the topics array to find the name of a resources parent topic
+	 * Once found it is assinged to the TopicItem variable
+	 * @param {{topics:array}} topics 
+	 * @param {{item:object}} item 
+	 */
+
+	const getTopicTitle = (topics, item) => {
+		topics.forEach((topic) => {
+			return topic.id === item.resource.parent ? TopicItem = topic.title : null;
+		
+		});
+	};
+
+	return (
+		<React.Fragment>
```

Comment:
React.Fragment is outdated. Please use `<>` and `</>` instead. [Check it out!](https://reactjs.org/docs/fragments.html#short-syntax)

Reply:
I changed it to  <> and </>. I saw it said some tools don't support it fully yet, do you think that might be an issue?